charts/airbyte/values.yaml

```diff
@@ -1,9 +1,110 @@
 ## @section Global Parameters
+
+## @param global.imageRegistry Global Docker image registry
 ##
 global:
-  ## @param global.imageRegistry Global Docker image registry
-  ##
   imageRegistry: ""
+  serviceAccountName: &service-account-name "airbyte-admin"
+  # configMapName: "airbyte-env"
```

Comment:
I don't understand why this on is commented out.  do we expect future users to use this key or can we kill it?

Reply:
We can kill it, just forgot to drop it out from the code